ex03.ex

```diff
@@ -59,7 +59,14 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def parity(true),  do: :even
+  def parity(false), do: :odd
+
```

Comment:
# -1

Why take to heads here? You just need the one to test, and then the tail:

~~~ elixir
def odd_even([]), do: []
def odd_even([ h | t ]), do: [ parity(is_even(h)) | odd_even(t) ]
~~~